### DISCUSSION TITLE
Ticket structure and understanding what is still "to be done" for the MVP

### DISCUSSION THREAD

**Discussion by da-kami** [get10101/10101#197] [General]
What @bonomat  mentioned in the retro yesterday about the state of our tickets made me think a bit. Here's a summary of my view on what is not captured in tickets yet:

- Orderbook: Logic and communication (stream) from orderbook to clients (i.e. app and maker)
- Coordinator: Some logic, we started integrating app and coordinator, but there are not tickets.
- Maker: Everything? 😅
- App features (also missing in dependent other systems):
    - Update / Close position
    - Perpetual position, i.e. rollovers (descope for initial MVP?)

---

I think it's better to create epics for `Features` rather than having epics per system; if we want e.g. rollovers as feature it will effect multiple systems. It might be better to capture a feature ticket that describes the feature for all the involved systems rather than having e.g. a mobile-rollover (...-rollover) ticket.
- We could update the existing (App) epics to describe overall features rather than focussing on the app (now that the main screens are there this makes more sense I think)

---

I think it would be good to capture these soon:

- Updating / closing a position. So far we mostly focus on creating the position, the next step will be updating / closing the position based on orders that are being created. We should capture that, potentially as one ticket that spans over all the involved systems for now.
- Perpetuals: how to come to a decision on when to trigger / accept a rollover and what role and responsibility do the different systems have
    - Perpetuals will be a bit tricky with the coordinator in the middle. If we assume that one trade may have a position open with multiple other traders, we will likely require some rollover-sync between the coordinator and the traders. This can become quite complex and I think it will require some more thought :)

**Comment by da-kami** [comment]
> We could update the existing (App) epics to describe overall features rather than focussing on the app (now that the main screens are there this makes more sense I think)

Example to make this more understandable:

For the `Trade Feature` epic I (unknowingly) already did that to some extent when I created it, see https://github.com/get10101/10101/issues/68

Pretty much all tickets attached to that epic will not only concern the mobile app, but also the other systems. 
However, this feature might still be cut on a level that is too high - we might want to have `Update / Close` position as *epic* and then have tickets attached to that for the different systems, with some tickets potentially spanning across systems.

**Comment by klochowicz** [comment]
is this discussion still relevant or shall we close it? @da-kami ?

**Comment by da-kami** [comment]
I think we already moved in that direction, but it will be good to keep it in mind. I believe thinking in overall features is generally better than focusing too much on the too specific system. Closing this.
